pretyping/coercion.ml

```diff
@@ -483,7 +483,10 @@ let inh_tosort_force ?loc env sigma j =
   with Not_found | NoCoercion ->
     error_not_a_type ?loc env sigma j
 
-let inh_coerce_to_sort ?loc env sigma j =
+let
+
+
+inh_coerce_to_sort ?loc env sigma j =
```

Comment:
I got notified because I'm a codeowner of this file. The change seems spurious ;-)

Reply:
Indeed, thanks.